react/Product.js

```diff
@@ -73,15 +73,22 @@ const formatGTIN = (gtin) => {
 const parseSKUToOffer = (
   item,
   currency,
-  { decimals, pricesWithTax, useSellerDefault }
+  { decimals, pricesWithTax, useSellerDefault, gtinValue }
 ) => {
   const seller = useSellerDefault
     ? getSellerDefault(item.sellers)
     : lowHighForSellers(item.sellers, { pricesWithTax }).low
 
+  const { low: lowSeller } = lowHighForSellers(item.sellers, { pricesWithTax })
   const availability = getSKUAvailabilityString(seller)
 
   const price = getFinalPrice(seller, getSpotPrice, { decimals, pricesWithTax })
+  const lowPrice = getFinalPrice(lowSeller, getSpotPrice, {
+    decimals,
+    pricesWithTax,
+  })
```

Comment:
Reuse price calculation when possible:

```suggestion
const price = getFinalPrice(seller, getSpotPrice, { decimals, pricesWithTax })
const lowPrice = seller === lowSeller
  ? price
  : getFinalPrice(lowSeller, getSpotPrice, {
    decimals,
    pricesWithTax,
  })
```

This avoids redundant calculations when `seller` and `lowSeller` are the same object.